### DISCUSSION TITLE
Can't sync with Mi Band 4/5

### DISCUSSION THREAD

**Discussion by Inkalika** [NightscoutFoundation/xDrip#3224] [Ideas]
Previously used the mod https://github.com/Pikernot/xDrip. It has support for Mi Band 4, 5 watches. In the latest version xDrip original, Mi Band 4 watches are not synchronized with the application.
Is it possible to add synchronization of these types of watches with xDrip original?

**Comment by Navid200** [comment]
MiBand is supported by a companion app:

WatchDrip

**Reply by Inkalika** [reply]
I used this program to set up the transmission of sugar in the clock, but in mod the transmission works, and the value of 5.1mmol is always transmitted through the original хdrip. Maybe I did something wrong, but I returned to the version-mod and watches began to take on the value of sugar again.

**Reply by Navid200** [reply]
I'm sorry I don't follow.  I see references to mod or version-mod, which have no meaning for me.

Please coordinate with WatchDrip developer.  If there is something that needs correction in xDrip, he is the person to do it.
The whole point of moving the MiBand support out of xDrip was to allow him to provide support to users better.
He has more experience about this than any other xDrip developer.
There is nothing we can do to xDrip without his involvement to address a MiBand issue.

**Reply by RonaldMigo** [reply]
Did you do this after you stared using watchdrip: 

 _If you had the watch previously configured in xDrip. Go to Settings -> Smart watch feature -> MiBand and disable “Use Miband” checkbox (this is an outdated feature)_

I took my a while to get it working stable but that was due to the fact that on the previous phone, the andriod version was to old (at least that was my assumption)
